HelpMyStreetFE/HelpMyStreetFE/Controllers/RegistrationController.cs

```diff
@@ -204,7 +204,14 @@ public async Task<ActionResult> StepThreePost([FromForm] StepThreeFormModel form
 
                 await _groupMemberService.AddUserToDefaultGroups(user.ID);
 
-                return Redirect("/account");
+                if (user.ReferringGroupId.HasValue)
+                {
+                    return Redirect($"/community/joinandgo/{user.ReferringGroupId.Value}");
```

Comment:
Sadly, I don't believe we actually need /joinandgo.  AddUserToDefaultGroups should already add the user to any ReferringGroup - so at this point we can just redirect to the group landing page.